app/src/main/java/org/simple/clinic/storage/migrations/Migration_58_59.kt

```diff
@@ -0,0 +1,25 @@
+package org.simple.clinic.storage.migrations
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+import org.simple.clinic.storage.inTransaction
+import javax.inject.Inject
+
+@Suppress("ClassName")
+class Migration_58_59 @Inject constructor() : Migration(58, 59) {
+
+  override fun migrate(database: SupportSQLiteDatabase) {
+    database.inTransaction {
+      execSQL("""
+        UPDATE "Patient" SET "syncStatus" = 'PENDING' WHERE "uuid" IN (
+          SELECT P.uuid FROM "Patient" P
+          LEFT JOIN (
+            SELECT * FROM "PatientAddress"
+              WHERE LENGTH("zone") > 0
```

Comment:
I just realised that the fields shown for India will never have Zone since it's a Bangladesh only field. This means that the value of this field will always be blank and this migration will end up forcing **ALL** patients to sync.

I will make the change to remove the zone from this check and force-push.

Reply:
@vinaysshenoy It is correct that it'll will be blank in India and that is expected. This query is only checking for length **greater** than zero which will be false for patients registered in India. This query is correct as it is. Please revert back to including the zone too?

Reply:
You... are right. Brain is fried. Shall revert. 😅

Reply:
This surfaces the problem... maybe this PR is missing a couple of tests too.

Reply:
Reverted. I also update the tests to cover all cases.